identifiers/country-kr/single_member_constituencies.csv

```diff
@@ -0,0 +1,253 @@
+id,name

```

Comment:
It looks like there's only 252 entries in this file, but the number of single-member districts is supposed to be 253, so one must be missing.

Reply:
Thanks! Found the missing one.